$NetBSD: patch-af,v 1.1.1.1 2006/12/19 22:54:51 joerg Exp $

Run-time path fix for Solaris for the X client libraries.  Without this
patch, libraries such as libXext.so will not contain a run-time link path
to /usr/pkg/xorg/lib, causing run time link failures or possibly linking
with other (non-Pkgsrc) X libraries on the system!

--- sun.cf.orig	2005-12-04 06:20:48.000000000 +1300
+++ sun.cf	2006-10-26 13:11:56.535497000 +1300
@@ -688,10 +688,10 @@
     /* This is to work around what appears to be a PIC bug in some gcc/egcs
        versions that shows up with the Solaris 7/x86 <ctype.h>. */
 #    define SharedLibraryLdCmd $(LD)
-#    define SharedLibraryLoadFlags -G -z textwarn
+#    define SharedLibraryLoadFlags -G -z textwarn LibraryRpathLoadFlags
 #   else
 #    define SharedLibraryLdCmd $(CC)
-#    define SharedLibraryLoadFlags -shared -z text
+#    define SharedLibraryLoadFlags -shared -z text LibraryRpathLoadFlags
 #   endif
 #  endif
 # else
